### ISSUE TITLE
Unable to start

### ISSUE THREAD

**Issue by withteja** [mr-pennyworth/alfred-better-dictionaries#25]
```
[19:29:16.304] Better Dictionaries[Keyword] Processing complete
[19:29:16.305] Better Dictionaries[Keyword] Passing output '' to Run Script
[19:29:16.531] ERROR: Better Dictionaries[Run Script] No matching processes belonging to you were found
[19:29:26.969] Better Dictionaries[Keyword] Processing complete
[19:29:26.970] Better Dictionaries[Keyword] Passing output '' to Run Script
[19:29:27.074] ERROR: Better Dictionaries[Run Script] xattr: ./AlfredExtraPane.app: No such xattr: com.apple.quarantine
xattr: ./alfred-dict-server: No such xattr: com.apple.quarantine
xattr: ./BetterDict: No such xattr: com.apple.quarantine
xattr: ./cocoaDialog.app: No such xattr: com.apple.quarantine
xattr: ./jq: No such xattr: com.apple.quarantine
No matching processes belonging to you were found
```

Encountering this when I ran 

`.setup-better-dictiona..` command

Device: Apple M2 ( MacOS  - 14.2.1 )

What I did?
* Downloaded `.alfredWorflow` from releases.
* Double clicked it and it took me to the alfred app and gave all the permissions (including the ones in Privacy & Setting)
* It didn't work so I deleted the workflow and did the `.setup-..` command. Did this multiple times.
* I didn't had logs for the first few times when I tried to setup, the logs above are from the last time.
* Finally I did reset factory command and it put `cocoaDialog` in my trash and when I did `.setup` I am seeing the issue.

**Comment by mr-pennyworth** [comment]
Thanks for the report @withteja!
Those logs are normal, so they shouldn't be an issue.

After running `.setup-better-dictionaries`, did you run `.dict-import`?
If not, I think doing that would solve any problems.
If you run into any problems while running `.dict-import`, we can figure out what's going on by looking at the logs :)

**Comment by showyoumagics** [comment]
When initializing, I encountered the same issue. When executing the dictionary import, I received the following error message:
<img width="849" alt="CleanShot 2024-03-15 at 17 37 44@2x" src="https://github.com/mr-pennyworth/alfred-better-dictionaries/assets/45767048/4ef5bff7-f58b-44e2-afe7-ac37c68f3769">

**Comment by showyoumagics** [comment]
I created the folder and encountered the following error when executing:
<img width="849" alt="CleanShot 2024-03-15 at 18 36 57@2x" src="https://github.com/mr-pennyworth/alfred-better-dictionaries/assets/45767048/103cba4d-e3b9-4dd1-8d17-e9b14b8e5eb3">

**Comment by WillSuo-Github** [comment]
same here

**Comment by MatiasFernandez** [comment]
Same issue here @mr-pennyworth, I get the same error on the initial setup, and importing dictionaries is not working for me. These are the logs:

``` [14:23:16.795] Better Dictionaries[Keyword] Processing complete
[14:23:16.804] Better Dictionaries[Keyword] Passing output '' to Run Script
[14:23:16.932] ERROR: Better Dictionaries[Run Script] xattr: ./AlfredExtraPane.app: No such xattr: com.apple.quarantine
xattr: ./alfred-dict-server: No such xattr: com.apple.quarantine
xattr: ./BetterDict: No such xattr: com.apple.quarantine
xattr: ./cocoaDialog.app: No such xattr: com.apple.quarantine
xattr: ./jq: No such xattr: com.apple.quarantine
No matching processes belonging to you were found
[14:23:23.149] Better Dictionaries[Script Filter] Queuing argument '(null)'
[14:23:33.318] Better Dictionaries[Script Filter] Script with argv '(null)' finished
[14:23:33.323] Better Dictionaries[Script Filter] {
  "items": [
    {
      "title": "Larousse Editorial Diccionario General de la Lengua Espa\u00f1ola",
      "arg": "/System/Library/AssetsV2/com_apple_MobileAsset_DictionaryServices_dictionaryOSX/f27147abfd9081073d6960e07997c8520d850003.asset/AssetData/Spanish.dictionary"
    },
    {
      "title": "New Oxford American Dictionary",
      "arg": "/System/Library/AssetsV2/com_apple_MobileAsset_DictionaryServices_dictionaryOSX/2187acba4774bd437214380aa79af5c5c6e34869.asset/AssetData/New Oxford American Dictionary.dictionary"
    },
    {
      "title": "Gran Diccionario Oxford - Espa\u00f1ol-Ingl\u00e9s \u2022 Ingl\u00e9s-Espa\u00f1ol",
      "arg": "/System/Library/AssetsV2/com_apple_MobileAsset_DictionaryServices_dictionaryOSX/b4984e6aa7cf66b9ba9c2a81e5b10807d14f40ef.asset/AssetData/Spanish - English.dictionary"
    },
    {
      "title": "Oxford American Writer\u2019s Thesaurus",
      "arg": "/System/Library/AssetsV2/com_apple_MobileAsset_DictionaryServices_dictionaryOSX/8d446327e477e192f31434968af30a1bd1c3a317.asset/AssetData/Oxford American Writer's Thesaurus.dictionary"
    }
  ]
}
[14:23:37.051] Better Dictionaries[Script Filter] Processing complete
[14:23:37.054] Better Dictionaries[Script Filter] Passing output '/System/Library/AssetsV2/com_apple_MobileAsset_DictionaryServices_dictionaryOSX/2187acba4774bd437214380aa79af5c5c6e34869.asset/AssetData/New Oxford American Dictionary.dictionary' to Run Script
[14:23:45.030] ERROR: Better Dictionaries[Run Script] Traceback (most recent call last):
  File "BetterDict.py", line 451, in <module>
  File "BetterDict.py", line 331, in import_dict
  File "BetterDict.py", line 362, in start_search_server
FileNotFoundError: [Errno 2] No such file or directory: '/Users/matyf/Library/Application Support/Alfred/Workflow Data/mr.pennyworth.betterDicts/db.log'
[68509] Failed to execute script 'BetterDict' due to unhandled exception!
[14:23:45.041] Better Dictionaries[Run Script] Processing complete
[14:23:45.042] Better Dictionaries[Run Script] Passing output '' to Post Notification```

**Comment by mr-pennyworth** [comment]
Thanks @showyoumagics, @WillSuo-Github, @MatiasFernandez, and @withteja for the detailed report!
I have identified the issue and fixed it: 6cf3c2aefcc4daee72d0b8c9a6a512c1addd9c77

Latest release: https://github.com/mr-pennyworth/alfred-better-dictionaries/releases/tag/0.2.9
